gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-container/src/main/java/io/gravitee/rest/api/standalone/jetty/JettyEmbeddedContainer.java

```diff
@@ -116,18 +119,18 @@ protected void doStart() throws Exception {
             throw new IllegalStateException("At least one API should be enabled");
         }
 
-        server.setHandler(new ContextHandlerCollection(contexts.toArray(new ServletContextHandler[contexts.size()])));
+        server.setHandler(new ContextHandlerCollection(contexts.toArray(new ServletContextHandler[0])));
```

Comment:
Is it better to initialize with 0 instead of the real size?

Reply:
Yes, can't recall the reason, I think it is going to be re-allocated anyhow, so better to don't do it.

Reply:
(Sonar warning)